downstream/modules/platform/proc-installing-the-ansible-builder-rpm.adoc

```diff
@@ -198,32 +198,32 @@ To upgrade between minor releases of AAP 2, use this general workflow.
 
 . Run `./setup.sh` to upgrade the installation.
 
-For example, to upgrade from version 2.1.2-1 to 2.2.0-7, make sure that both setup bundles are on the initial controller node where the installation occurred:
+For example, to upgrade from version 2.2.0-7 to 2.3-1.2, make sure that both setup bundles are on the initial controller node where the installation occurred:
 
 ----
     $ ls -1F
-ansible-automation-platform-setup-bundle-2.1.2-1/
-ansible-automation-platform-setup-bundle-2.1.2-1.tar.gz
 ansible-automation-platform-setup-bundle-2.2.0-7/
 ansible-automation-platform-setup-bundle-2.2.0-7.tar.gz
+ansible-automation-platform-setup-bundle-2.3-1.2/
+ansible-automation-platform-setup-bundle-2.3-1.2.tar.gz
 ----
 
-Back up the 2.1.2-1 installation:
+Back up the 2.2.0-7 installation:
 ----
-$ cd ansible-automation-platform-setup-bundle-2.1.2-1
+$ cd ansible-automation-platform-setup-bundle-2.2.0-7
 $ sudo ./setup.sh -b
 $ cd ..
 ----
 
-Copy the 2.1.2-1 inventory file into the 2.2.0-7 bundle directory:
+Copy the 2.2.0-7 inventory file into the 2.3-1.2 bundle directory:
 ----
-$ cd ansible-automation-platform-setup-bundle-2.1.2-1
-$ cp inventory ../ansible-automation-platform-setup-bundle-2.2.0-7/
+$ cd ansible-automation-platform-setup-bundle-2.2.0-7
+$ cp inventory ../ansible-automation-platform-setup-bundle-2.3-1.2/
```

Comment:
should there be a difference between the formats for 2.2 and 2.3? it looks like 2.3-x.y and 2.2.x.y

Reply:
Hi Ian! I noticed this difference, too, but these are the version numbers provided to me by the reporter.

Reply:
OK. Do you want it merging?

Reply:
Hi Ian, yes please! Thank you!